libinitializer/RPCInitializer.cpp

```diff
@@ -78,14 +78,14 @@ void RPCInitializer::initConfig(boost::property_tree::ptree const& _pt)
             auto blockChain = m_ledgerManager->blockChain(it);
             auto channelRPCServer = std::weak_ptr<dev::ChannelRPCServer>(m_channelRPCServer);
             auto handler = blockChain->onReady([groupID, channelRPCServer](int64_t number) {
-                LOG(TRACE) << "Push block notify: " << (int)groupID << "-" << number;
+                LOG(INFO) << "Push block notify: " << (int)groupID << "-" << number;
                 auto c = channelRPCServer.lock();
 
                 if (c)
                 {
                     std::string topic =
                         "_block_notify_" + boost::lexical_cast<std::string>((int)groupID);
-                    std::string content = boost::lexical_cast<std::string>(groupID) + "," +
+                    std::string content = boost::lexical_cast<std::string>((int)groupID) + "," +
```

Comment:
use GROUP_ID instead of int

Reply:
fixed in: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/527/files#diff-fc72f03b22cdddcf4af335bee13f73e6